docs/source/common/finetuning_example.rst

```diff
@@ -0,0 +1,76 @@
+To use a Task for finetuning:
+
+1. Load your data and organize it using a DataModule customized for the task (example: :class:`~flash.vision.ImageClassificationData`).
+2. Choose and initialize your Task which has state-of-the-art backbones built in (example: :class:`~flash.vision.ImageClassifier`).
+3. Init a :class:`flash.core.trainer.Trainer`.
+4. Choose a finetune strategy (example: "freeze") and call :func:`flash.core.trainer.Trainer.finetune` with your data.
+5. Save your finetuned model.
+
+|
+
+Here's an example of finetuning.
+
+.. testcode:: finetune
```

Comment:
```suggestion
.. testcode:: python
```
not sure but I thought that this for prog. language annotation...

Reply:
my bad, you are right :] https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html#directive-testcode